mypy/messages.py

```diff
@@ -253,14 +253,17 @@ def format_simple(self, typ: Type, verbosity: int = 0) -> str:
                 base_str = itype.type.fullname()
             else:
                 base_str = itype.type.name()
-            if itype.args == []:
+            if itype.args == [] or len(itype.args) == 1 and type(itype.args[0]) == AnyType:
```

Comment:
On second thought, yeah let me remove this code change, and update the error messages instead.